tests/recipes/test_full_finetune_distributed.py

```diff
@@ -91,6 +99,13 @@ def test_loss(
             cmd.append(f"fsdp_sharding_strategy={fsdp_sharding_strategy}")
         model_config = MODEL_TEST_CONFIGS[model_type]
         cmd = cmd + self._get_test_config_overrides() + model_config
+        # "optimizer_in_bwd=True" would free gradient info before clip_grad, causing
+        # wrong grad_norm, so we only test one of them each time. But loss values
+        # should be the same. torch >= 2.5 required to enable optimizer_in_backward.
+        if not optim_in_bwd:
+            cmd.append("clip_grad_norm=100")
+        elif optim_in_bwd and version.parse(torch.__version__).base_version < "2.5.0":
```

Comment:
`optimizer_in_backward` needs torch version to be no less than 2.5, shall I add the version check here?